.github/workflows/power9.yml

```diff
@@ -0,0 +1,118 @@
+name: github-POWER9
+
+# Only allow manual runs until at2 runners are available.
+on: workflow_dispatch
+  #pull_request:
+  #  paths-ignore:
+  #  - '**/*.rst'
+  #  - '**/*.md'
+  #  - '**/requirements.txt'
+  #  - '**/*.py'
+  #  - 'docs/**'
+  #  types: [ opened, reopened, synchronize ]
+
+permissions:
+  contents: none
+
+# Cancels any in progress 'workflow' associated with this PR
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  PR_POWER9_GCC930_OPENMP_SERIAL_LEFT_REL:
+    name: PR_POWER9_GCC930_OPENMP_SERIAL_LEFT_REL
+    runs-on: [kk-env-gcc-9.3.0-latest]
+
+    strategy:
+      matrix:
+        include:
+          - backend: "SERIAL"
+            serial: "ON"
+            openmp: "OFF"
+          - backend: "OPENMP"
+            serial: "OFF"
+            openmp: "ON"
+          - backend: "SERIAL_OPENMP"
+            serial: "ON"
+            openmp: "ON"
+    
+    steps:
+      - name: checkout_kokkos_kernels
+        uses: actions/checkout@v3
+        with:
+          path: kokkos-kernels
+
+      - name: checkout_kokkos
+        uses: actions/checkout@v3
+        with:
+          repository: kokkos/kokkos
+          ref: ${{ github.base_ref }}
+          path: kokkos
+
+      - name: configure_kokkos
+        run: |
+          mkdir -p kokkos/{build-${{ matrix.backend }},install-${{ matrix.backend }}}
+          cd kokkos/build-${{ matrix.backend }}
+          GPP=$(which g++)
+          cmake -DCMAKE_CXX_COMPILER=$GPP \
+            -DCMAKE_CXX_FLAGS="-O3 -Wall -Wunused-parameter -Wshadow -pedantic -Werror -Wsign-compare -Wtype-limits -Wignored-qualifiers -Wempty-body -Wclobbered -Wuninitialized" \
+            -DCMAKE_EXE_LINKER_FLAGS= \
+            -DCMAKE_INSTALL_PREFIX=$PWD/../install-${{ matrix.backend }} \
+            -DKokkos_ENABLE_SERIAL=${{ matrix.serial }} \
+            -DKokkos_ENABLE_OPENMP=${{ matrix.openmp }} \
+            -DKokkos_ARCH_PASCAL60=ON \
```

Comment:
Actually I do not think that we still have Pascal build or that we support this architecture at all?
Shouldn't this be V100?

Reply:
No, same as above WRT cm_test_all_sandia.